pkgs/by-name/se/sequoia-sq/package.nix

```diff
@@ -10,22 +10,23 @@
   capnproto,
   installShellFiles,
   openssl,
+  cacert,
   sqlite,
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "sequoia-sq";
-  version = "1.1.0";
+  version = "1.3.0";
 
   src = fetchFromGitLab {
     owner = "sequoia-pgp";
     repo = "sequoia-sq";
-    rev = "v${version}";
-    hash = "sha256-m6uUqTXswzdtIabNgijdU54VGQSk0SkSqdh+7m1Q7RU=";
+    tag = "v${version}";
```

Comment:
Let's put this change, along with the `finalAttrs:` change in a separate commit please.

Reply:
I put the change in revised commit https://github.com/NixOS/nixpkgs/commit/097ad426efb6a3f6cd197fc9d5e03239cf6741b8 🙏